spec/functional/resource/cron_spec.rb

```diff
@@ -113,7 +113,7 @@ def cron_should_not_exists(cron_name)
     end
   end
 
-  exclude_solaris = %w{solaris opensolaris solaris2 omnios}.include?(ohai[:platform])
+  exclude_solaris = %w{solaris solaris2 omnios}.include?(ohai[:platform])
```

Comment:
same

Reply:
I think we can skip these just because we're in functional and that's never gonna happen